litebird_sim/bandpasses.py

```diff
@@ -0,0 +1,275 @@
+from dataclasses import dataclass
+
+import numpy as np
+from scipy import signal
+
+from typing import Union
+from uuid import UUID
+
+from .imo import Imo
+
+
+@dataclass
+class BandPassInfo(object):
+
+    """A class wrapping the basic information about a detector bandpass.
+    This  class   encodes the basic properties of a frequency band.
+    It can be initialized in three ways:
+    - Through the default constructor, will assume top-hat bandpasses
+    provided the band centroid and the width are issued.
+    - Through the class method :meth:`.from_imo`, which reads the
+      detector bandpass from the LiteBIRD Instrument Model (see
+      the :class:`.Imo` class).
+
+    Args:
+        - nu_center (float):  center frequency in GHz
+        - bandwidth (float):  width of the band (default=0.2)
+        - range_offset (int): increase the freq. range by an offset (default +- 10 from the edges)
```

Comment:
The parameter `range_offset` is not clear: is it an integer because it specifies the number of samples to add to `nsamp`? (It does not seem so, but I confess I do not completely understand the logic.)

Reply:
Yeah you are right it is a bit awkward parameter, 
`range offset` essentially defines a frequency range  it must be slightly  wider  than the bandwidth so that we are able include the apodization profile .